### ISSUE TITLE
issue with lock file creation

### ISSUE THREAD

**Issue by jreps** [OHDSI/OhdsiRTools#37]
I used this package to create a lock file for renv yesterday on a windows machine. 

To reproduce add the latest DeepPatientLevelPrediction then write: `OhdsiRTools::createRenvLockFile(rootPackage = 'DeepPatientLevelPrediction',includeRootPackage = T)` 

for some reason it put DeepPatientLevelPrediction as "RemoteUsername" : "ohdsi-studies" where it should be "ohdsi" .  I think this is due to includeRootPackage = T assuming the package is in OHDSI-studies

**Comment by schuemie** [comment]
Yes, I always assumed the root package would be a study package, not a HADES package. For now please manually fix the lock file. I'll think of a solution (that will probably require manually specifying the root package is not a study package).
